solidity/contracts/OracleDev.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity 0.4.24;
+
+import "./Oracle.sol";
+import "./interfaces/LinkExInterface.sol";
+
+contract OracleDev is Oracle {
+
+  mapping(bytes32 => address) public priceFeeds;
```

Comment:
Can we make the mapping `bytes32 => LinkExInterface` and take the hit on casting when we set instead of when we get?